doc/overview.rst

```diff
@@ -65,6 +65,12 @@ subject_indices : list
 disp_files : bool
     Display status.
 
+.. note:: Anywhere a ``dict`` is supported as an option (e.g.,
+          ``mf_prebad`` or ``proj_nums``), a special entry ``'__default__'``
+          can be used turn the dictionary into a
+          :class:`~python:collections.defaultdict` instance.
+          This is useful in cases where a single set of values works for most
+          subjects, but a few need different ones.
```

Comment:
I think here it could be clearer that the special entry should be `__default__: [MEG8888, MEG9999]` or similar.

Reply:
EDIT: I see now it actually should be `__default__: [8888, 9999]` (without the "MEG" prefix)